h/static/scripts/features.js

```diff
@@ -1,90 +1,34 @@
 /**
  * Feature flag client.
  *
- * This is a small utility which will periodically retrieve the application
```

Comment:
Good job on removing outdated docs, but it would be nice to replace them with a one or two line summary of what this does actually now do :)